src/ORM/Queries/SQLConditionalExpression.php

```diff
@@ -163,12 +171,20 @@ public function addLeftJoin($table, $onPredicate, $tableAlias = '', $order = 20,
      * @return $this Self reference
      */
     public function addInnerJoin($table, $onPredicate, $tableAlias = null, $order = 20, $parameters = [])
+    {
+        return $this->addJoin($table, 'INNER', $onPredicate, $tableAlias, $order, $parameters);
+    }
+
+    /**
+     * Add a JOIN criteria
+     */
+    private function addJoin($table, $type, $onPredicate, $tableAlias = null, $order = 20, $parameters = []): static
```

Comment:
This method wasn't strictly necessary, but it seems appropriate to reduce code duplication by moving the repeated code into this new method.
I can see a use case for this being public, but that's not the purpose of this PR. We can always do taht as a separate future enhancement down the road.